packages/react-composites/src/composites/common/MeetingConferencePhoneInfoModal.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+import React from 'react';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { MeetingConferencePhoneInfoModal, ConferencePhoneInfo } from '@internal/react-components';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { TeamsMeetingAudioConferencingDetails } from '@azure/communication-calling';
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/** @private */
+export const MeetingConferencePhoneInfo = (props: {
+  conferencePhoneInfoList: ConferencePhoneInfo[];
+  showMeetingConferencePhoneInfoModal: boolean;
+  onDismissMeetingPhoneInfoSettings: () => void;
+}): JSX.Element => {
+  return (
+    <MeetingConferencePhoneInfoModal
+      conferencePhoneInfoList={props.conferencePhoneInfoList}
+      showModal={props.showMeetingConferencePhoneInfoModal}
+      onDismissMeetingPhoneInfoSettings={props.onDismissMeetingPhoneInfoSettings}
+    />
+  );
+};
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/** @private */
+export const convertConferencePhoneInfo = (
+  meetingConferencePhoneInfo?: TeamsMeetingAudioConferencingDetails
+): ConferencePhoneInfo[] => {
+  if (!meetingConferencePhoneInfo) {
+    return [];
+  }
+
+  const convertedPhoneInfo = [];
+
+  for (let i = 0; i < meetingConferencePhoneInfo.phoneNumbers.length; i++) {
+    const phoneNumber = meetingConferencePhoneInfo.phoneNumbers[i];
+    if (phoneNumber.tollPhoneNumber) {
```

Comment:
avoid this if, everything is the same inside the blocks except for isTollFree, instead just have
```ts
     convertedPhoneInfo.push({
        phoneNumber: phoneNumber.tollPhoneNumber.phoneNumber,
        conferenceId: meetingConferencePhoneInfo.phoneConferenceId,
        isTollFree: !phoneNumber.tollPhoneNumber,
        country: phoneNumber.countryName,
        city: phoneNumber.cityName
      });
```